drivers/hmis/app/models/hmis/enrollment_access_summary.rb

```diff
@@ -0,0 +1,43 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# backed by a db view
+module Hmis
+  class EnrollmentAccessSummary < ApplicationRecord
+    self.table_name = 'hmis_user_enrollment_activity_log_summaries'
+    self.primary_key = 'id'
+
+    belongs_to :user, class_name: 'Hmis::User'
+    belongs_to :enrollment, class_name: 'Hmis::Hud::Enrollment'
+
+    def readonly?
+      true
+    end
+
+    def self.apply_filter(user:, starts_on: nil, search_term: nil)
```

Comment:
Can you add a filter for Project IDs? This would be particularly helpful in trying to figure out whether a user has accessed any enrollments at a project that they shouldn't have (if they were granted access in error, for example).

Reply:
Yes I've added it. It was bit complicated though; to avoid having to fetch down all enrollment ids for a filtered project, I added the project_id on to the join table, which is now populated in the activity job

Reply:
Ok, thanks!